modules/pushsnap_scriptworker/files/requirements.txt

```diff
@@ -16,29 +13,28 @@ idna==2.7
 idna_ssl==1.1.0
 json-e==2.7.0
 jsonschema==2.6.0
+libnacl==1.3.6
```

Comment:
We probably need to add a ` # puppet: nodownload` comment here -- libnacl downloads seem to barf due to required system libs that aren't on puppet masters.

Reply:
It looks like `libnacl` is a dependency of `pymacaroons`. Newer versions of that use `pynacl`, which has wheels available. If we could upgrade to that, it would avoid the trouble here.

Reply:
Thanks for the context! I would prefer to avoid updating until the push-snap task gets back to green. pushsnapscript has a very dirty hack where we only import a part of the dependencies of snapcraft. The reasons of this hack are:

* snapcraft has some Ubuntu-specific deps and we're running it on CentOS
* there is no snapcraft package on CentOS 6.
* we can't run the snap package of snapcraft because CentOS 6 is not supported.

If I manage to get the next job, I plan to comment `requirements.txt` to explain why some deps aren't up to date. What do you guys think?

Reply:
Sure. However, not having the `# puppet: nodownload` is causing us to get puppet mail multiple times an hour, with the error message `Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-download-34e2ruc8/libnacl/`. Could you update your environment with that comment, at least, to reduce the mail errors?

Reply:
Oh, I'm sorry for the breakage. I didn't know personal env were subject to this kind of checks. This was fixed 5 hours ago.